spec/features/end_to_end/school_direct_salaried_journey_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "school-direct-salaried end-to-end journey", type: :feature do
+  background { given_i_am_authenticated }
+
+  scenario "submit for TRN", "feature_routes.school_direct_salaried": true do
```

Comment:
I'm wondering whether we need to have a scenario for the JS version? Seems like an overkill just to test the schools picker is working but it's considered as part of the main flow

Reply:
Scratch that, it WILL be overkill as we already have another spec covering this.